style.css

```diff
@@ -0,0 +1,169 @@
+@media only screen and (min-width: 768px) {
+  
+  
+}
+@media only screen and (min-width: 375px) {
+  
+  
+}
+body{
+min-height: 100vh;
+}
+
+.headline{
+display: flex;
+flex-direction: column;
+align-items: flex-start;
+padding: 0px 24px;
+gap: 12px;
+position: absolute;
+width: 375px;
+height: 604px;
+left: calc(50% - 375px/2);
+top: 92px;
+background-image: url(Images/header-shapes-mobile.png);
+background-repeat: no-repeat;
+}
+
+.header{
+width: 335px;
```

Comment:
- [ ] Kindly remove all fixed widths as here from your containers that must stretch upon different screen widths. You should use percentages(%) rather than pixels for the best CSS practices.